integration/tests/basic/test_failed_transactions.py

```diff
@@ -24,3 +25,19 @@ def test_bump_allocator_out_of_memory_expected_error(self):
             assert resp["status"] == 0
         except ValueError as exc:
             assert "Error: memory allocation failed, out of memory." in exc.args[0]["message"]
+
+    def test_send_non_neon_token_without_chain_id(self, account_with_all_tokens, web3_client_sol, sol_price, operator):
```

Comment:
Please, move it to test_payment_in_different_tokens.py, add to md file (where tests are described), add multipletokens mark, and change try-exept block to pytest.raise

Reply:
Done